drivers/uktty/ns16550/ns16550.c

```diff
@@ -83,6 +88,10 @@
 static __u8 ns16550_uart_initialized;
 static __u64 ns16550_uart_base;
 
+#if CONFIG_LIBUKLIBPARAM
+UK_LIBPARAM_PARAM_ALIAS(base, &ns16550_uart_base, __u64, "ns15550 base");
```

Comment:
How does someone know about this? Should there maybe be a kconfig hint?

Reply:
Good question. In theory every library should export a `README.md` and these should be automatically collected along with doxygen to autogenerated documentation. Also before https://github.com/unikraft/unikraft/pull/1471 `libuklibparam` printed usage info upon parse error or coming across the parameter `help`. Of course with early cmdline this creates a circular dependency with console, but perhaps we could keep a flag in ukboot and print these after the console is initialized, at least until we implement a console buffer. Assuming successful console initialization, that is. I'll push an update in https://github.com/unikraft/unikraft/pull/1471.

Reply:
Yes this makes sense. This could be a nice track after we have the ISR plugin work in. We could extend that to check additional things such as this. Especially once we will have KASLR in and we need a clearly defined section for relocations before/after KASLR.